scripts/cmake/vcpkg_execute_build_process.cmake

```
@@ -42,6 +42,7 @@ set(Z_VCPKG_EXECUTE_BUILD_PROCESS_RETRY_ERROR_MESSAGES
     "LINK : fatal error LNK1104:"
     "LINK : fatal error LNK1201:"
     "ld terminated with signal 9"
+    "Killed signal terminated program cc1plus"
```

```suggestion
    "Killed signal terminated program"
```

might as well allow things other than cc1plus to be killed; for example, we might hit `Killed signal terminated program cc1`, in a C project (although admittedly that's much less common)